docs/user/bots.rst

```diff
@@ -1908,6 +1908,26 @@ Use this command to create/update the database and reload the bot:
 
 .. _intelmq.bots.experts.deduplicator.expert:
 
+Domain valid
+^^^^^^^^^^^^
+
+**Information**
+
+* `name:` `intelmq.bots.experts.domain_valid.expert`
+* `lookup:` no
+* `public:` yes
+* `cache (redis db):` none
+* `description:` Valid first level domain from TLD list `tlds-alpha-by-domain.txt`. Latest list: https://data.iana.org/TLD/
+
+**Configuration Parameters**
+   * `domain_field`- event domain field
+   * `tlds_domains_list` - local file location '/opt/intelmq/var/lib/bots/domain_valid/tlds-alpha-by-domain.txt'
+
+**Description**
+Bot validating domain in event, if domain is not valid than event is dropped.
```

Comment:
```suggestion
If the field given in `domain_field` does not exist in the event, the event is dropped.
If the domain contains underscores (``_``), the event is dropped.
If the domain is not valid according to the `validators library <https://pypi.org/project/validators/>`_, the event is dropped.
If the domain's last part (the TLD) is not in the TLD-list configured by parameter ``tlds_domains_list``, the field is dropped.
```
Being more verbose what the bot actually does. This documentation is intended for users, so that they don't need to look at the code to figure out what a component is doing.